observations/obs_converters/GSI2DART/work/quickbuild.csh

```diff
@@ -5,58 +5,139 @@
 # http://www.image.ucar.edu/DAReS/DART/DART_download
 #
 # compile all converter programs
+#
+# usage: [ -mpi | -nompi ]
+#
+# environment variable options:
+#  before running this script, do:
+#   "setenv CODE_DEBUG 1" (csh) or "export CODE_DEBUG=1" (bash)
+#  to keep the .o and .mod files in the current directory instead of
+#  removing them at the end.  this usually improves runtime error reports
+#  and these files are required by most debuggers.
+#----------------------------------------------------------------------
+
+# programs which have the option of building with MPI:
+
+set MPI_TARGETS = "gsi_to_dart"
+
+# set default (override with -mpi or -nompi):
+#  0 = build without MPI, 1 = build with MPI
+
+set with_mpi = 1
```

Comment:
This follows the style of any 'model' quickbuild ... they all support the specification of targets that should be compiled with MPI ... If you find any that do not, we need to change them for consistency.